From 47fabd70379f7e832766bf8b265f00dd939573d7 Mon Sep 17 00:00:00 2001
From: Danny Lin <danny@kdrag0n.dev>
Date: Sat, 9 Oct 2021 14:39:38 -0700
Subject: [PATCH] overlay: SystemUI: Follow light/dark theme in power menu

Now that we've modified the power menu to refresh on UI mode changes,
make it follow the system light/dark theme for better integration in
light mode.

Requires frameworks/base commit:
    SystemUI: Always refresh global actions dialog on UI mode change

Change-Id: I4290ab4c0b50a37c6d111e9c72a3f11267c7172a
Signed-off-by: althafvly <althafvly@gmail.com>
---
 .../SystemUI/res/values-night/colors.xml      | 22 +++++++++++++++++++
 .../packages/SystemUI/res/values/colors.xml   | 22 +++++++++++++++++++
 2 files changed, 44 insertions(+)
 create mode 100644 overlay/common/frameworks/base/packages/SystemUI/res/values-night/colors.xml
 create mode 100644 overlay/common/frameworks/base/packages/SystemUI/res/values/colors.xml

diff --git a/overlay/common/frameworks/base/packages/SystemUI/res/values-night/colors.xml b/overlay/common/frameworks/base/packages/SystemUI/res/values-night/colors.xml
new file mode 100644
index 00000000..af390339
--- /dev/null
+++ b/overlay/common/frameworks/base/packages/SystemUI/res/values-night/colors.xml
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+    Copyright (C) 2022 The LineageOS Project
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+<resources>
+    <!-- Dark Mode for power menu -->
+    <color name="global_actions_lite_background">@*android:color/primary_device_default_dark</color>
+    <color name="global_actions_lite_button_background">@*android:color/surface_dark</color>
+    <color name="global_actions_lite_text">@*android:color/foreground_device_default_dark</color>
+</resources>
diff --git a/overlay/common/frameworks/base/packages/SystemUI/res/values/colors.xml b/overlay/common/frameworks/base/packages/SystemUI/res/values/colors.xml
new file mode 100644
index 00000000..306952e7
--- /dev/null
+++ b/overlay/common/frameworks/base/packages/SystemUI/res/values/colors.xml
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+    Copyright (C) 2022 The LineageOS Project
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+<resources>
+    <!-- Light Mode for power menu -->
+    <color name="global_actions_lite_background">@*android:color/primary_device_default_light</color>
+    <color name="global_actions_lite_button_background">@*android:color/surface_light</color>
+    <color name="global_actions_lite_text">@*android:color/foreground_device_default_light</color>
+</resources>
-- 
2.36.1

